python/cudnn/__init__.py

```
@@ -391,12 +391,26 @@ def _load_optional_symbol(name: str) -> Any:
     globals()[name] = value
     return value
 
 
+def _optional_dependency_message(name: str, error: Exception) -> str:
+    # A DSL that is installed but below the floor must not be reported as a
+    # missing dependency: "pip install [cutedsl]" would change nothing.
+    try:
+        from .frost.buffers import cutedsl_requirement_error
+
+        too_old = cutedsl_requirement_error(name)
+    except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)